### PR TITLE
Remove unnecessary reshape

### PR DESCRIPTION
**Pull request by gab-chen**
*Description of changes:*
Please let me know if you need more details. 
I tested removing the `.reshape((-1, 1))` from the PyTorch version of the notebook. The train loss and validation loss remain the same after the removal. But I'm unsure how the code changed here translates to other framework versions like JAX or TF. Can you double check if the change will work for non-PyTorch versions?

Removal of the unnecessary `reshape` helps ease the burden on the learners at this early chapter of the book. 

By submitting this pull request, I confirm that you can use, modify,
copy, and redistribute this contribution, under the terms of your
choice.

### CODE REVIEW

**Comment by gab-chen** [comment]
I skimmed through the error trace. The errors occurred at .jax files. Does that mean the change is incompatible with JAX only?

**Comment by astonzhang** [comment]
Thanks. At this point we'll keep it for compatibility with other frameworks with shared code cells.
